src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+  </head>
+
+  <body>
+    <div data-qa="card" class="card">
+      <img
+        class="card__logo"
+        src="./images/imac.jpeg"
+        alt="images"
+      >
+
+      <h2 class="card__title">
+        <a
+          class="card__link"
+          href="#"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+      </h2>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviuw"> Reviews: 5 </div>
+      </div>
+
+      <div class="card__value">
+        <div class="card__price">Price:</div>
+        <div class="card__cost"> $2,199</div>
+      </div>
+
+      <a
+          data-qa="hover"
+          class="card__buy"
+          href="#"
+        >
+        Buy
+        <button class="card__button" >
+        </button>
```

Comment:
I think here <a closing part not on the same line 
should be
<a
>
and button also moved a little bit rigth